src/Client/KeenIOClient.php

```diff
@@ -49,6 +55,7 @@ class KeenIOClient extends GuzzleClient
      * @param array $config
      *
      * @returns \KeenIO\Client\KeenIOClient
+     * @return KeenIOClient
```

Comment:
you should fix the `@returns` line to `@return` instead of adding a new line without removing the broken one (and this should probably be separate from adding support for access keys anyway)

Reply:
fixed